src/ngraph/runtime/cpu/builder/convolution.cpp

```diff
@@ -341,7 +349,8 @@ namespace ngraph
                                 node, args, out);
```

Comment:
These kernels if (!mkldnn), also don't have arena. do they need an arena?

Reply:
reference kernels dont run in arenas for now.